org.chrisoft.qmidiplayer.yml

```diff
@@ -0,0 +1,76 @@
+id: org.chrisoft.qmidiplayer
+runtime: org.kde.Platform
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+command: qmidiplayer
+
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --filesystem=host
+  - --device=dri
+
+modules:
+  # Visualization
+  - name: glfw
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -DBUILD_SHARED_LIBS=ON
+      - -DGLFW_BUILD_EXAMPLES=OFF
+      - -DGLFW_BUILD_TESTS=OFF
+      - -DGLFW_BUILD_DOCS=OFF
+    sources:
+      - type: archive
+        url: https://github.com/glfw/glfw/releases/download/3.3.8/glfw-3.3.8.zip
+        sha256: 4d025083cc4a3dd1f91ab9b9ba4f5807193823e565a5bcf4be202669d9911ea6
+    cleanup:
+      - /include
+      - /lib/pkgconfig
+
+  - shared-modules/glew/glew.json
+
+  - shared-modules/glu/glu-9.json
+
+  - name: devil
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+    subdir: DevIL
+    builddir: true
+    sources:
+      - type: archive
+        url: https://sourceforge.net/projects/openil/files/DevIL/1.8.0/DevIL-1.8.0.tar.gz
+        sha256: 0075973ee7dd89f0507873e2580ac78336452d29d34a07134b208f44e2feb709
+
+  # Minimum QMidiPlayer build
+  - shared-modules/linux-audio/fluidsynth2.json
+
+  - name: rtmidi
+    buildsystem: cmake-ninja
+    sources:
+      - type: archive
+        url: https://github.com/thestk/rtmidi/archive/refs/tags/6.0.0.tar.gz
+        sha256: ef7bcda27fee6936b651c29ebe9544c74959d0b1583b716ce80a1c6fea7617f0
+        x-checker-data:
+          type: anitya
+          project-id: 4220
+          url-template: https://github.com/thestk/rtmidi/archive/refs/tags/$version.tar.gz
+
+  - name: qmidiplayer
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -DBUILD_VISUALIZATION=ON
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DUSE_QT6=OFF
```

Comment:
why not using Qt6? There is a runtime for that.

Reply:
I don't have a Qt 6 flatpak runtime and failed to download the runtime/Sdk package previously due to network issue. (it's pretty slow to access flathub in my current region). Will do another attempt later. Will this be a blocking issue for creating the flathub repo?

Reply:
Tried again and now I have Qt 6.6 runtime locally. Patch updated :)